### DISCUSSION TITLE
clr-interop generics

### DISCUSSION THREAD

**Discussion by furesoft** [rzubek/CSLisp#21] [Ideas]
may add a constructor call with generics possible something like:

`(make-clr-instance 'System.List (of 'string)`

ohh and add 

`(get-clr-type instance)` function

**Comment by rzubek** [comment]
Good idea!

Sometime over the next few days I'll start a page to collect all of the improvement ideas.

**Reply by rzubek** [reply]
Moved to discussions

**Reply by furesoft** [reply]
What is the current state of the feature?
